### PR TITLE
add Idris Effect library

### PR DESCRIPTION
**Pull request by lemastero**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,6 +42,9 @@ in various programming languages.
 * **F\***: an ML-like effectful dependently typed functional programming language aimed at program verification  
   ([www](https://fstar-lang.org))
 
+* **Idris Effects**: library for algebraic effects and handlers in Idriss - inspired by Eff language  
+  ([github](https://github.com/idris-lang/Idris-dev/tree/master/libs/effects))  ([www](https://www.idris-lang.org/documentation/effects/))
```
Comment (yallop) [inline README.md]: I think the "www" link is sufficient here: the GitHub link points to the middle of the source tree of a larger project, which isn't particularly enlightening.

File: README.md [hunk 2]
```diff
@@ -42,6 +42,9 @@ in various programming languages.
 * **F\***: an ML-like effectful dependently typed functional programming language aimed at program verification  
   ([www](https://fstar-lang.org))
 
+* **Idris Effects**: library for algebraic effects and handlers in Idriss - inspired by Eff language
```
[the remark this answers is not in the document]
Comment (lemastero) [reply README.md]: Done: https://github.com/yallop/effects-bibliography/pull/56/commits/a6f15b559c009ca7f8436dda6227757a5e8d68e9

File: README.md (hunk 1 above)
Comment (lemastero) [reply README.md]: Done - github link removed: https://github.com/yallop/effects-bibliography/pull/56/commits/42a6d2520a7c1d865a360664bc2b150895a082d5

**Comment by yallop** [comment]
Thank you!

**Pull request merged**